components/create/CreateCollection.vue

```diff
@@ -26,6 +26,7 @@
       <NeoField
         :label="`${$t('mint.collection.name.label')} *`"
         required
+        data-testid="collection-name"
         :error="!name">
```

Comment:
@roiLeo where to insert a data-testid specific for the warning?

Reply:
You don't need to, just check if `o-field__message-danger` is present neer this input

Btw it's a test that is not too relevant since it's html that will check if input is present or not & you won't be able to submit form

Reply:
idk who wrote the code but `required` prop on `NeoField` is useless and don't work

Reply:
when i wrote i thought about a case when dangers messages were not being displayed. Will leave only `o-field__message-danger` ty!